job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/views/ListViewSpec.groovy

```diff
@@ -310,59 +310,128 @@ class ListViewSpec<T extends ListView> extends Specification {
 
         where:
         filter || children
-        { ->
-            status {
-                matchType(INCLUDE_UNMATCHED)
-                status(Status.UNSTABLE, Status.STABLE, Status.ABORTED, Status.FAILED, Status.DISABLED)
-            }
-        } | [
+                { ->
```

Comment:
sorry for this but I use the default IntelliJ like is defined in contributor's guide.
You want I revert indent modifications ?